### DISCUSSION TITLE
DAI 2024-2025 - My picocli project - Nicolas Carbonara

### DISCUSSION THREAD

**Discussion by Syseria** [heig-vd-dai-course/heig-vd-dai-course#267] [Show and tell]
https://github.com/Syseria/DAI-04

**Comment by ludelafo** [comment]
Everything looks good to me except the following lines in the `pom.xml` file: https://github.com/Syseria/DAI-04/blob/main/pom.xml#L9-L11.

You must keep the initial values to ensure Maven can compile the project with the right Java version and encodings (more on this in the next chapter we'll see together).
